nautobot_ssot_infoblox/tests/fixtures_infoblox.py

```diff
@@ -9,7 +9,7 @@
 
 FIXTURES = os.environ.get("FIXTURE_DIR", "nautobot_ssot_infoblox/tests/fixtures")
 
-LOCALHOST = os.environ.get("TEST_LOCALHOST_URL", "http://localhost:4440/wapi/v2.12")
+LOCALHOST = os.environ.get("TEST_LOCALHOST_URL", "https://localhost:4440/")
```

Comment:
This change might be why the tests are now failing, you dropped the `/wapi/v2.12` from this variable.

Reply:
The tests were failing prior to my making these changes FYI. This particular change was made though to specify HTTPS as the connection should always be using that to CloudVision. Also, the LOCALHOST variable is used for the URL, being passed to the InfobloxApi class. The WAPI variable is set by NAUTOBOT_INFOBLOX_WAPI_VERSION env var and defaults to v2.12 in the nautobot_config file already. I do see that the other test request URLs need to be updated to account for that though.

Reply:
This has been reverted to include the wapi portion and hardset https now. The tests are passing now.